src/main/java/com/box/sdk/BoxFileVersionRetention.java

```diff
@@ -67,6 +67,7 @@ public BoxFileVersionRetention(BoxAPIConnection api, String id) {
      * and {@link BoxRetentionPolicyAssignment#getFileVersionsUnderRetention(String...)} instead.
      * Retrieves all file version retentions matching given filters as an Iterable.
      */
+    @Deprecated
```

Comment:
Is there any alternative to this method? Is it possible to point a client to it?

Reply:
Yes, but it is documented in JavaDoc above the method:

```
* @deprecated This method will be deprecated in the future. Please use
* {@link BoxRetentionPolicyAssignment#getFilesUnderRetention(int, String...)}
* and {@link BoxRetentionPolicyAssignment#getFileVersionsUnderRetention(String...)} instead.
```